src/main/java/HereApi/IncidentsJsonParser.java

```diff
@@ -0,0 +1,139 @@
+package HereApi;
+
+import Decoder.HereDecoder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import openlr.location.Location;
+import openlr.map.Line;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.StringReader;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+public class IncidentsJsonParser {
```

Comment:
Javadoc ergänzen. Kann ähnlich sein, wie in getAffectedLines()

Reply:
gemacht